packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -259,5 +263,13 @@ export const styles = css`
                 background-color: rgba(${borderRgbPartialColor}, 0.07);
             }
         `
+    ),
+    themeBehavior(
```

Comment:
I was actually working on this same bug (you didn't claim the issue), and I think it's a little more complicated to address.
1. The comment for token `sectionBackgroundColor` says it is for dialogs and menus, but it isn't used anywere. I think we should be changing this token to say it is used for menus and dropdowns, change the values to what we want, then actually use the token. We have been using `applicationBackgroundColor`, which is really not the right token to use.
2. Using a transparent background color for menus and dropdowns (in the color theme) is bad, because the background shows through. We ought to compute a reasonable opaque equivalent (i.e. assuming a ForestGreen background) and use that instead.
3. We also use a transparent value (for all three themes) for `popupBorderColor`, and it looks kind of shitty when it overlays something. You can see this in the matrix story. I think we should take the same approach of making a reasonable opaque equivalent.

If you agree with my points, I can take over this bug if you want. The only part I think is iffy is point 3 -- if we leave it transparent, the border of the listbox will always match the border of the input (for outline appearance). If we make it opaque, it won't necessarily match, because the input border is transparent, too -- ugh.